Cargo.toml

```diff
@@ -13,6 +13,7 @@ keywords = ["tracing", "stackdriver", "logging"]
 status = "actively-developed"
 
 [dependencies]
+chrono = "0.4.19"
```

Comment:
down the lane, when we decide to switch to `time` as many crates as currently doing, there is some work to do for `rfc3339` formatting. initially, i started on that route https://github.com/yottabytt/tracing-stackdriver/commit/7017f75b146c009b3a1be27e78f72eaae28b0710 , but thought i can continue further after discussing abt the `time` vs `chrono` and our take on it here.

Reply:
> Is fixed number of digits a thing in the RFC?

it looked pretty on the log lines at least for me. https://datatracker.ietf.org/doc/html/rfc3339#section-5.1 touches abt it.